src/main/java/uk/gov/hmcts/reform/cpo/repository/CasePaymentOrderQueryFilter.java

```diff
@@ -11,36 +17,51 @@
 @Getter
 public class CasePaymentOrderQueryFilter {
 
-    public static final String CASES_ID = "caseId";
-    public static final String ORDER_REFERENCE = "orderReference";
-
     private Integer pageSize;
     private Integer pageNumber;
-    private List<String> listOfIds;
-    private List<String> listOfCasesIds;
+    private List<String> cpoIds;
+    private List<String> caseIds;
 
 
-    public boolean isACasesIdQuery() {
-        return !listOfCasesIds.isEmpty();
+    public boolean isFindByCaseIdQuery() {
+        return !caseIds.isEmpty();
     }
 
     public boolean isAnIdsQuery() {
-        return !listOfIds.isEmpty();
+        return !cpoIds.isEmpty();
     }
 
-    public boolean isAnIdsAndCasesIdQuery() {
-        return isACasesIdQuery() && isAnIdsQuery();
+    public boolean isFilterByBothIdAndCaseId() {
+        return isFindByCaseIdQuery() && isAnIdsQuery();
     }
 
-    public boolean isItAnEmptyCriteria() {
-        return listOfIds.isEmpty() && listOfCasesIds.isEmpty();
+    public boolean noFilters() {
+        return cpoIds.isEmpty() && caseIds.isEmpty();
     }
 
     public List<UUID> getListUUID() {
-        return listOfIds.stream().map(UUID::fromString).collect(Collectors.toList());
+        return cpoIds.stream().map(UUID::fromString).collect(Collectors.toList());
     }
 
     public List<Long> getListOfLongCasesIds() {
-        return listOfCasesIds.stream().map(Long::parseLong).collect(Collectors.toList());
+        return caseIds.stream().map(Long::parseLong).collect(Collectors.toList());
+    }
+
+    public PageRequest getPageRequest() {
+        final List<Sort.Order> orders = new ArrayList<>();
+        orders.add(new Sort.Order(Sort.Direction.ASC, CasePaymentOrderEntity.CASE_ID));
+        orders.add(new Sort.Order(Sort.Direction.ASC, CasePaymentOrderEntity.ORDER_REFERENCE));
+        return PageRequest.of(
+            this.getPageNumber(),
+            this.getPageSize(),
+            Sort.by(orders)
+        );
```

Comment:
can we use some some simpler syntax to configure sorting like:
```
Pageable pageable = PageRequest.of(0, 5, Sort.by(
    Order.asc("name"),
    Order.desc("id")));
```
https://reflectoring.io/spring-boot-paging/

Reply:
DONE